pages/api/v1/communityTab/getAllUsers.ts

```
@@ -0,0 +1,99 @@
+		};
+	}
+};
+
+const handler: NextApiHandler<UsersResponse | MessageType> = async (req, res) => {
+	storeApiKeyUsage(req);
+	const network = String(req.headers['x-network']);
+	if (!network || !isValidNetwork(network)) return res.status(400).json({ message: 'Invalid Network' });
+
+	const { page = 1, username, sortOption } = req.body;
+
+	if (isNaN(page)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)